package/boot/uboot-c3000/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME=u-boot
+PKG_VERSION:=2020.07-rc5
```

Comment:
the customer use this version for stable. they don't use the last version. I see that many u-boots are not using the latest version on Openwrt.

Reply:
You only have very few patches in U-Boot, it looks like most of your base code is upstream which is good. OpenWrt uses outdated U-Boot version for SoC with bad U-Boot with many thousand lines of ugly patches. This U-Boot support looks much better and if it is not so hard it would be good if you upgrade to a recent U-Boot version like 2022.04.

Reply:
thanks huake! 
I am trying to update the patchs. I think I need to cancel this pull request, and summit a new one. Thank you for your understanding. This is my first pull request.

Reply:
If you're going to update your patches, please do not create another new PR, try force push instead.
https://openwrt.org/docs/guide-developer/frequent-pr-mistakes#do_not_multi-post_or_re-post_prs

Reply:
Hi mans0n, sorry for inconvenient, there is a miss aligned in our internal team, we have close this request and focus on the mail list patch. Thanks, Yang.